server/runtime/src/main/resources/schema/infinispan-server-14.0.xsd

```diff
@@ -956,6 +956,36 @@
       </xs:attribute>
    </xs:complexType>
 
+   <xs:complexType name="masked-credential">
+      <xs:annotation>
+         <xs:documentation>
+            A masked credential using the format `MASKED_VALUE;SALT;ITERATION`
+         </xs:documentation>
+      </xs:annotation>
+      <xs:attribute name="masked" type="xs:string" use="required">
+         <xs:annotation>
+            <xs:documentation>
+               The masked credential.
+            </xs:documentation>
+         </xs:annotation>
+      </xs:attribute>
+   </xs:complexType>
+
+   <xs:complexType name="command-credential">
+      <xs:annotation>
+         <xs:documentation>
+            A masked credential using the format `MASKED_VALUE;SALT;ITERATION`
```

Comment:
it is not a masked credential but a command :)

Reply:
Fixed